src/ResetViewControl.ts

```diff
@@ -14,15 +14,22 @@ export type ResetViewControlOptions = {
    * It renders a box by default.
    */
   icon?: string;
+
+  /**
+   * "zoom" and "center" properties are useful when the map is mounted in a particular position 
+   * and you need this button to reset the view to another position.
+  */
+  zoomToReset?: number;
```

Comment:
Can you document these with a comment per property, like `title` and `icon`?

Reply:
Done